### DISCUSSION TITLE
[Feedback] Old is Gold

### DISCUSSION THREAD

**Discussion by VaibhavSys** [razonyang/hugo-theme-bootstrap#670] [General]
Hey, I think this is a great theme for hugo and I really like this theme however, I think it was way better in around [Feb 4, 2022](https://github.com/razonyang/hugo-theme-bootstrap/commits/master?after=76f13ea62659b8a3bcb3db2733a3dec97a1bf45d+664&branch=master&qualified_name=refs%2Fheads%2Fmaster).
It was much more clean and minimal at that time, while being full of features, and the build times have increasd A LOT by then.
Awesome theme!

**Comment by razonyang** [comment]
Thanks for using this theme. I changed the theme a lot since `v0.67.0`, in order to bring more features and make the theme much more flexible. Admittedly, this does make the build step complicated and the build time longer, since we use the Hugo Pipes instead of pre-builds. I think it's still worth it, such as it allows users to customize and use any Bootstrap components they want, which is not possible with pre-built resources.
